.github/workflows/testing-and-deployment.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
```

Comment:
If you remove the explicit install of the wheel in setup.py, we could move this to the `include` section and use another identifier for vtk-version and handle it in the `Setup vtk` step.

Reply:
Good point. Implemented.